public/index.html

```diff
@@ -59,9 +59,9 @@
 <!--<br>-->
 <!--<br>-->
 <!--<br>-->
-<!--<web-info-widget area="GER" language="de" debug="true"></web-info-widget>-->
-<script async src='/build/embed.js'></script>
-<!--<script async src='/build/widget.js'></script>-->
+<!--<coverified-widget debug="true"></coverified-widget>-->
```

Comment:
instead of leaving dead code here you could just remove it?!

Reply:
`index.html` is only used for local development and preview links. Having these comments allows me to switch between different usage modes in dev more easily. I'd like to keep tham as they have no other impact.